### PR TITLE
Add prerequisites to contribute

### PR DESCRIPTION
**Pull request by opcoach**

### CODE REVIEW

**Comment by opcoach** [comment]
@BeckerWdf If you have 2 minutes to review, you can also ;)

File: Prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+|--------|-----------|-------------------------------------------------------------|---------------------------------------------------|
+| Action | Necessary | Install an Eclipse IDE and the project to contribute to (see Part 1)    |[![Eclipse FAQ](https://img.shields.io/badge/Eclipse%20FAQ-Launch%20Configuration-blue?logo=eclipse)](https://github.com/eclipse-platform/eclipse.platform/blob/master/docs/FAQ/FAQ_What_is_a_launch_configuration.md) |
+
+
+# Technical knowledge
+
+| Type          | Level       | Description                                      | How to do it?                                      |
+|---------------|-------------|--------------------------------------------------|---------------------------------------------------|
+| Skill | Essential | Add a bookmark to reference this FAQ !         |    [![Consult the Eclipse FAQ](https://img.shields.io/badge/Eclipse%20FAQ-The%20Official%20FAQ-8A2BE2?logo=eclipse)](https://github.com/eclipse-platform/eclipse.platform/blob/master/docs/FAQ/The_Official_Eclipse_FAQs.md#The-Eclipse-Community)    |
```
Comment (BeckerWdf) [inline Prerequisites.md]: whats the difference of "Necessary" (above) and "Essential" (here)?

File: Prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+| Action | Necessary | Fill and Sign the Eclipse Contributor Agreement (ECA)       | [![Read and Sign ECA](https://img.shields.io/badge/Read%20and%20Sign-ECA-blue?logo=eclipse&style=flat-square)](https://www.eclipse.org/legal/ECA.php/#related-links)  |
+| Action | Necessary | Create or use a GitHub account                              | [![Sign up](https://img.shields.io/badge/Create%20Account-GitHub-black?logo=github&style=flat-square)](https://github.com/signup) |
+| Action | Necessary | Be aware of the different projects | [![Projects Guide](https://img.shields.io/badge/Projects%20Guide-Eclipse%20IDE-blue?logo=eclipse)](https://github.com/eclipse-ide/.github/blob/main/projects.md) |
+| Action | Necessary | Be aware of assistance to help you during your contribution  | [![Assistance](https://img.shields.io/badge/Ask_for_assistance-red?style=for-the-badge&logo=eclipseide)](https://github.com/eclipse-ide/.github/blob/main/help.md) |
```
Comment (BeckerWdf) [inline Prerequisites.md]: This step tells the contributors that they can get some help. So for me this is not an "action" that is "necessary" but rather something that is "good to know".

**Comment by BeckerWdf** [comment]
> @BeckerWdf If you have 2 minutes to review, you can also ;)

Sure

File: Prerequisites.md (hunk 1 above)
Comment (opcoach) [reply Prerequisites.md]: Necessary for the action means mandatory. One must create or use GitHub account.  I will change it.

**Review by BeckerWdf — APPROVED** [review]

**Pull request merged**